pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -160,6 +161,7 @@ func NewInstallerController(
 
 		installerPodImageFn: getInstallerPodImageFromEnv,
 		clock:               clock.RealClock{},
+		installerBackOff:    backOffDuration(10*time.Second, 1.5, 10*time.Minute),
```

Comment:
it means we will retry ~10 times, which seems reasonable

Reply:
Yes. Unchanged btw.

We start with 10, then 15, then 22.5 up to 10min.